docs/whats_new.rst

```diff
@@ -18,13 +20,16 @@ Bug fixes
 
 Documentation
 ~~~~~~~~~~~~~
+Adds Storing simulation results on a file section in the getting started page.
 
 Performance
 ~~~~~~~~~~~
+Exporting outputs as netCDF4 is much faster than exporting a pandas DataFrame, especially for large models.
```

Comment:
This should be a list, starting with "- "

Note see that in "migration" branch I have updated the what's new section to add information about the developer that made the changes. Therefore, could you add your username and link to GH profile after each item you have developed?

Reply:
Done